assets/js/search.js

```diff
@@ -1,5 +1,7 @@
 let fuse
 
+const algolia = algoliasearch("27CIRMYZIB", "7e88305c04e90188508daa6c89e5f4df").initIndex("codemagic_docs");
```

Comment:
These are perfectly fine to be public, see the [InstaSearch.js](https://codesandbox.io/embed/github/algolia/doc-code-samples/tree/master/InstantSearch.js/getting-started) demo.